packages/server/scripts/generate-subgraph-schema.ts

```diff
@@ -0,0 +1,18 @@
+import 'reflect-metadata';
```

Comment:
What is it used for?
If only CI, I'd replace it with codegen CLI command, with flag to an alternative, most simple config to generate `schema.graphql` and do the CI checks. No need to build the entire app